src/main/java/seedu/address/model/recipe/Recipe.java

```diff
@@ -105,6 +105,28 @@ public String getFullRecipe() {
         return String.format("%d. %s\n%s%s", this.getId(), this.name, ingredients, steps.toString().stripTrailing());
     }
 
+    /**
+     * Returns the ingredients needed in textual format.
+     */
+    public String getIngredientsText() {
+        StringBuilder ingredients = new StringBuilder();
+        for (Ingredient ingredient : ingredientList) {
+            ingredients.append(ingredient.getName()).append(" ").append(ingredient.getQuantity()).append("\n");
```

Comment:
Just nitpicking but can probably split into various lines to improve readability